math/sieve_of_eratosthenes.cpp

```diff
@@ -1,32 +1,27 @@
-/*
- * Sieve of Eratosthenes is an algorithm to find the primes 
+/**
+ * Sieve of Eratosthenes is an algorithm to find the primes
  * that is between 2 to N (as defined in main).
  *
- * Time Complexity  : O(N * log N)
- * Space Complexity : O(N)
- */
+ * Time Complexity: O(N * log N)
+ * Space Complexity: O(N)
+**/
 
 #include <iostream>
-using namespace std;
 
 #define MAX 10000000
 
 int isprime[MAX];
```

Comment:
instead of defining this much large array, use dynamic memory allocation to do the same task

Reply:
@ayaankhan98 I have implemented this algorithm in #821 in a much-optimized way because of this #820